src/process_utility.c

```diff
@@ -277,6 +277,8 @@ process_create_foreign_server_start(ProcessUtilityArgs *args)
 				 errmsg("operation not supported for a TimescaleDB data node"),
 				 errhint("Use add_data_node() to add data nodes to a "
 						 "TimescaleDB distributed database.")));
+
+	return false;
```

Comment:
I am confused by the change in return value here and in many functions below. Most or all of them just seem to always return `false` instead of `void` (nothing), and it is unclear what `false` semantically means.

I would at suggest to at least create an `enum` for the return value (e.g., DDL_DONE, DDL_CONTINUE) or similar. This would make it a lot more clear what the intention of the return value is.

Reply:
Agree, makes sense. I am also a bit confused, but since we have most functions defined this way I expected to make them equal in definition. I would be happy to change it in another PR if it is ok